src/ManifoldMuseum.jl

```diff
@@ -166,13 +218,21 @@ exp!(M::Manifold, y, x, v) = error("Exponential map not implemented on a $(typeo
 Exponential map of tangent vector `t*v` at point `x` from manifold `M`.
 """
 function exp(M::Manifold, x, v)
-    x2 = similar_result(M, x, v)
-    exp!(M, x2, x, v)
-    return x2
+    y = similar_result(M, exp, x, v)
+    exp!(M, y, x, v)
+    return y
 end
 
 exp(M::Manifold, x, v, t) = exp(M, x, t*v)
 
+"""
+    exp(M::Manifold, x, v, T::AbstractVector)
+
+Exponential map of tangent vector `t*v` at point `x` from manifold `M` for
+each `t` in `T`.
+"""
+exp(M::Manifold, x, v, T::AbstractVector) = [exp(M, x, v, t) for t in T]
```

Comment:
Do you need to define this? With broadcasting, wouldn't the user simply write `exp.(M, x, v, T)` for an `AbstractVector` of real numbers? By the way, maybe it makes sense to restrict the `t` (or `T`) arguments to `<:Real`?

Reply:
> Do you need to define this? With broadcasting, wouldn't the user simply write `exp.(M, x, v, T)` for an `AbstractVector` of real numbers?

In some cases, it's significantly more efficient to compute the whole trajectory at once than to broadcast, the prime example being the numerically integrated exponential maps.

>By the way, maybe it makes sense to restrict the `t` (or `T`) arguments to `<:Real`?

Yeah, I think so. I'll change it.

Reply:
But this is not achieved by what is written here?
```julia
[exp(M, x, v, t) for t in T]
```
doesn't compute one geodesic and then evaluates at all `t in T`, but it re-computes the exponential map for every `t`.

Reply:
Yes, this is just a suitable default that will be fine for most manifolds but which developers are free to specialize for their manifold if there's a more efficient approach. `MetricManifold` does just that when `HasMetric` is not defined (See https://github.com/JuliaNLSolvers/ManifoldMuseum/pull/12/files#diff-53108f2d38bac2c2779a8604bf2fc16fR307)

Reply:
Wouldn't `map(geodesic(M, x, v), T)` be a more reasonable default here? In case someone overloads `geodesic` with a more efficient implementation?

Reply:
But now that I'm looking at it more closely, a better default is to use `map`, since it (tries to) respect the type of `T` and is more efficient.